pkg/hintrunner/zero/zerohint_others.go

```diff
@@ -147,6 +148,53 @@ func createMemEnterScopeHinter(resolver hintReferenceResolver, memset bool) (hin
 	return newMemEnterScopeHint(value, memset), nil
 }
 
+// GetFeltBitLength hint assigns to `bit_length` the bit length of `x` variable
+//
+// `newGetFeltBitLengthHint` takes 2 operanders as arguments
+//   - `x` is a felt variable
+//   - `bit_length` is the variable that will store the bit length of x
+func newGetFeltBitLengthHint(x, bitLength hinter.ResOperander) hinter.Hinter {
```

Comment:
could you add at least one unittest for this?

Reply:
sure

Reply:
done!